image-specs/el7.3-base

```diff
@@ -0,0 +1,14 @@
+#base=libguestfs:centos-7.3
+#name=el7.3-base
+#osinfo=centos7.3
+#distro=el7
+#version=v1
+#arch=x86_64
+#expand=/dev/sda2
+#lvexpand=/dev/cl/root
+#root-partition=/dev/cl/root
+root-password password:123456
+update
+install qemu-guest-agent
+uninstall iwl*
```

Comment:
Looks like uninstall *firmware* is even better, but let's go with this one and improve later.
Anything preventing from commit?